pkg/antctl/command_list.go

```diff
@@ -34,8 +35,12 @@ type commandList struct {
 }
 
 func (cl *commandList) applyPersistentFlagsToRoot(root *cobra.Command) {
+	defaultPath := os.Getenv("KUBECONFIG")
+	if len(strings.TrimSpace(defaultPath)) == 0 {
+		defaultPath = clientcmd.RecommendedHomeFile
+	}
 	root.PersistentFlags().BoolP("verbose", "v", false, "enable verbose output")
-	root.PersistentFlags().StringP("kubeconfig", "k", clientcmd.RecommendedHomeFile, "absolute path to the kubeconfig file")
+	root.PersistentFlags().StringP("kubeconfig", "k", defaultPath, "absolute path to the kubeconfig file")
```

Comment:
no strong opinion but my preference would be to use empty (`""`) as the default here, and have all the logic in  `resolveKubeconfig`. What do you think?

Reply:
I'm good about this. I thought users could know the default path if we could show it in path, but looks kubectl shows the default kubeconfig value as empty string.
I will update the implementation.